### ISSUE TITLE
Will the NPC vehicle type be updated

### ISSUE THREAD

**Issue by liuppppp** [lgsvl/simulator#780]
I see now that there are pedestrian and NPC vehicle systems, whether you will support more diverse types, such as bicycles, motorcycles, etc

thanks.

**Comment by EricBoiseLGSVL** [comment]
@liuppppp Yes, we are currently working on NorthAmerican, Japanese vehicles, bicycles, trailer trucks and motorcycles.  We hope to release soon
